src/pybind/mgr/dashboard/controllers/nvmeof.py

```diff
@@ -1,15 +1,27 @@
 # -*- coding: utf-8 -*-
-from typing import Optional
+import logging
+from typing import Any, Dict, Optional
 
+from .. import mgr
 from ..model import nvmeof as model
 from ..security import Scope
 from ..tools import str_to_bool
-from . import APIDoc, APIRouter, Endpoint, EndpointDoc, Param, ReadPermission, RESTController
+from ..services.orchestrator import OrchClient
+from . import APIDoc, APIRouter, UIRouter, Endpoint, \
+    EndpointDoc, Param, ReadPermission, BaseController, RESTController
+
+logger = logging.getLogger()
```

Comment:
so the name of the module/file will be used
```suggestion
logger = logging.getLogger(__name__)
```